python/ray/autoscaler/aws/example-multi-node-type.yaml

```diff
@@ -16,27 +16,26 @@ available_node_types:
     cpu_4_ondemand:
         node_config:
             InstanceType: m4.xlarge
-        resources: {"CPU": 4}
+        # For AWS instances, autoscaler will automatically add the available
+        # CPUS/GPUS/accelerator_type ({"CPU": 4} for m4.xlarge) in "resources".
+        resources: {"Custom": 1}
         min_workers: 1
         max_workers: 5
     cpu_16_spot:
         node_config:
             InstanceType: m4.4xlarge
             InstanceMarketOptions:
                 MarketType: spot
-        resources: {"CPU": 16, "Custom1": 1}
         max_workers: 10
     gpu_1_ondemand:
         node_config:
             InstanceType: p2.xlarge
-        resources: {"CPU": 4, "GPU": 1, "Custom2": 2, "accelerator_type:K80": 1}
         max_workers: 4
         worker_setup_commands:
             - pip install tensorflow-gpu  # Example command.
     gpu_8_ondemand:
         node_config:
             InstanceType: p3.8xlarge
-        resources: {"CPU": 32, "GPU": 4, "accelerator_type:V100": 1}
```

Comment:
Can you actually keep this here but commented out? (and say-- note that the autoscaler auto fills this).